.github/workflows/pytest.yaml

```
@@ -6,7 +6,7 @@ on:
 jobs:
   pytest:
     runs-on: ubuntu-latest
-    container: ghcr.io/autowarefoundation/autoware-universe:latest
+    container: ghcr.io/autowarefoundation/autoware-universe:humble-latest
```

This sentence does not have to be modified because `latest` supports humble
```suggestion
    container: ghcr.io/autowarefoundation/autoware-universe:latest
```

---

Thank you for pointing out!
I applied your suggestion.